packages/fluentui/react-northstar/test/specs/commonTests/implementsShorthandProp.tsx

```diff
@@ -66,5 +69,27 @@ export default ((Component: React.ComponentType) => {
         });
       });
     });
+
+    if (options.implementsPopper) {
+      describe('implements all positioning props for Popper', () => {
+        Object.keys(positioningProps).forEach((positioningProp: keyof PositioningProps) => {
```

Comment:
nit: `Object.entries` since you also use value in a couple of places, but it's not tragic :D

Reply:
And there was another place similar to this one.